docs/contributing/style-guide.md

```diff
@@ -1,7 +1,405 @@
+
+| Variable | Description |
+| ----------- | ----------- |
+|id|A brief string that uniquely identifies the page within its parent folder. The id and the name of the file should be the same.|
+|title|The main title of the page. This value will automatically be rendered using the H1 style at the top of the page.
+|sidebar_label|This is what will appear in the left hand navigation tree for the page.|
+|description (optional)|This is what appears when the page is referenced in a Google search result.|
+
+### Code Example
+
+```markdown
+---
+id: style-guide
 title: Style Guide Introduction
-sidebar_label: Overview
+sidebar_label: Style Guide
+description: The Sauce Labs Documentation Style Guide
 ---
+```
+
+## Introduction
+
+The first paragraph of the documentation should set the user's expectations for what they will find on the page. 
+Describe the key benefits to the user, but do not include links.
+
+## Headers
+
+For accessibility and SEO reasons, never have an H4 header that isn't under an H3 header, 
+or an H3 header that isn't under an H2 header.
+
+* H1 headers should never be used in a document since the title is automatically generated as an H1.
+* H2 headers are used for SEO, so make sure they succinctly represent what a user will find on the page in that section
+* H3 headers are included in the page's table of contents on the right, so make sure the title describes something
+a user might want to click on
+* H4 headers are to emphasize things within a subsection of the page; these can be longer than the other headers if needed. 
+
+### Code Example
+
+```markdown
+## H2 Header
+### H3 Header
+#### H4 Header
+```
+
+## Content
+
+All words are rendered in the same paragraph even with line breaks, so long as there isn't an empty line.
+As such, it is good practice for each line to be less than 120 characters long for readability, when possible.
+
+### Code Example
+
+```markdown
+This
+will
+all
+be
+on
+one
+line.
+
+The empty line above creates a new paragraph.
+
+This forces a soft return<br />
+rather than creating a new paragraph
+```
+
+### Display
+
+This
+will
+all
+be
+on
+one
+line.
+
+The empty line above creates a new paragraph.
+
+This forces a soft return<br />
+rather than creating a new paragraph
+
+## Images
+
+All image files should be included in the `sauce-docs/static/img` directory, in a sub-directory that 
+corresponds to the referencing directory. (e.g., images for a document in the `sauce-docs/docs/contributing` directory 
+would be located in the `sauce-docs/static/img/contributing` directory.
+
+To add an image from that directory, you need to import a special method by placing this line
+below the [Frontmatter](#frontmatter), but above the [Introduction](#introduction)
+
+```markdown
+import useBaseUrl from '@docusaurus/useBaseUrl';
+```
+
+and then reference the image as follows:
+```markdown
+<img src={useBaseUrl('img/contributing/my-image.png')} alt="All images should have alt text" width="250"/>
+```
```

Comment:
This is correct, but this is actually HTML, not markdown, and is in conflict with the markdown image ref I commented on in the markdown-styles file of this review.